File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/Istat.ts
import {IntervalloTemporale} from '../anagrafiche/IntervalloTemporale';

export class Istat {
  reg: string = '000';
  prov: string;
  com: string;
  cap: string;
  localita: string;
  comuni_prov: string;
  validita: IntervalloTemporale;
  codiceBelfiore: string;
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/metaschema/Istat.cs
using AgronicaCoreModelsSTD.anagrafiche;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.metaschema
{
    public class Istat
    {

        public string reg { get; set; } = "000";
        public string prov { get; set; }
        public string com { get; set; }
        public string localita { get; set; }
        public string comuni_prov { get; set; }
        public string cap { get; set; }        

        public IntervalloTemporale validita { get; set; }

        /// <summary>
        /// è il codice del comune come appare sulle penultime 4 posizioni del codice fiscale (prima del carattere di controllo), es: D704 = Forlì
        /// </summary>
        public string codiceBelfiore { get; set; }

        public Istat()
        {

        }
    }
}
